### PR TITLE
refactoring-drainer-redundant-code

### PR DESCRIPTION
**Pull request by vijaygupta18**

### CODE REVIEW

File: src/EulerHS/KVConnector/DBSync.hs [hunk 1]
```diff
@@ -139,57 +113,49 @@ getDeleteQuery tag timestamp dbName deleteCommand deleteCommandV2 mappings = A.o
...
+  [ "contents" .= whereClauseToJson whereClause
   , "tag" .= ((T.pack . pascal . T.unpack) model <> "Options")
   ]
 
-getDbDeleteCommandJsonWithPrimaryKey :: forall be table. (HasCallStack, KVConnector (table Identity), Model be table, MeshMeta be table, A.ToJSON (table Identity)) => DBCommandVersion' -> Text -> table Identity -> Where be table -> A.Value
-getDbDeleteCommandJsonWithPrimaryKey version model table whereClause = A.object
-  [ "contents" .= ((whereClauseJsonWithPrimaryKey @be) version table $ whereClauseToJson version whereClause)
+getDbDeleteCommandJsonWithPrimaryKey :: forall be table. (HasCallStack, KVConnector (table Identity), Model be table, MeshMeta be table, A.ToJSON (table Identity)) => Text -> table Identity -> Where be table -> A.Value
+getDbDeleteCommandJsonWithPrimaryKey model table whereClause = A.object
+  [ "contents" .= ((whereClauseJsonWithPrimaryKey @be)  table $ whereClauseToJson  whereClause)
```
Comment (roman-bodavskiy) [inline src/EulerHS/KVConnector/DBSync.hs:124]: redundant spaces everywhere

File: src/EulerHS/KVConnector/Flow.hs [hunk 2]
```diff
@@ -451,7 +450,7 @@ updateObjectRedis meshCfg updVals setClauses addPrimaryKeyToWhereClause whereCla
           mkUpdateCmd version = if addPrimaryKeyToWhereClause
                         then getDbUpdateCommandJsonWithPrimaryKey version (tableName @(table Identity)) setClauses obj whereClause
                         else getDbUpdateCommandJson version (tableName @(table Identity)) setClauses whereClause
-          qCmd      = getUpdateQuery (pKeyText <> shard) time meshCfg.meshDBName (mkUpdateCmd V1') (mkUpdateCmd V2) (getTableMappings @(table Identity)) updatedModel
+          qCmd      = getUpdateQuery (pKeyText <> shard) time meshCfg.meshDBName (mkUpdateCmd V2) (getTableMappings @(table Identity)) updatedModel
```
Comment (roman-bodavskiy) [inline src/EulerHS/KVConnector/Flow.hs]: `mkUpdateCmd V1` do not used, so version not required here

File: src/EulerHS/KVConnector/Flow.hs [hunk 3]
```diff
@@ -1105,10 +1104,10 @@ deleteObjectRedis meshCfg addPrimaryKeyToWhereClause whereClause obj = do
       shard     = getShardedHashTag pKeyText
       pKey      = fromString . T.unpack $ pKeyText <> shard
       mkDeleteCmd version = if addPrimaryKeyToWhereClause
-                    then getDbDeleteCommandJsonWithPrimaryKey version (tableName @(table Identity)) obj whereClause
-                    else getDbDeleteCommandJson version (tableName @(table Identity)) whereClause
+                    then getDbDeleteCommandJsonWithPrimaryKey  (tableName @(table Identity)) obj whereClause
+                    else getDbDeleteCommandJson  (tableName @(table Identity)) whereClause
 
-      qCmd      = getDeleteQuery (pKeyText <> shard) time meshCfg.meshDBName (mkDeleteCmd V1') (mkDeleteCmd V2) (getTableMappings @(table Identity))
+      qCmd      = getDeleteQuery (pKeyText <> shard) time meshCfg.meshDBName (mkDeleteCmd V2) (getTableMappings @(table Identity))
```
Comment (roman-bodavskiy) [inline src/EulerHS/KVConnector/Flow.hs]: the same, `version` not required

File: src/EulerHS/KVConnector/Types.hs [hunk 4]
```diff
@@ -36,10 +36,8 @@ import Sequelize.SQLObject (ToSQLObject)
 ------------ TYPES AND CLASSES ------------
 
 -- toJSON V1 = [] this is how aeson works, so if we add V2 here, it will be not backward compatible
```
Comment (roman-bodavskiy) [inline src/EulerHS/KVConnector/Types.hs]: this comment can be removed now, if you have two values `V1` and `V2`

**Comment by roman-bodavskiy** [comment]
can you please test locally how it works with drainer

**Review by roman-bodavskiy — DISMISSED** [review]

**Review by roman-bodavskiy — DISMISSED** [review]

**Review by arjuna92 — APPROVED** [review]

**Pull request merged**